site/ja/tutorials/text/word_embeddings.ipynb

```diff
@@ -0,0 +1,665 @@
+    "\n",
+    "テキストデータセットを最初からロードする方法については、[テキスト読み込みのチュートリアル](../load_data/text.ipynb)を参照してください。"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "cQrYYEvvUuV7"
+   },
+   "outputs": [],
+   "source": [
+    "(train_data, test_data), info = tfds.load(\n",
+    "    'imdb_reviews/subwords8k', \n",
+    "    split = (tfds.Split.TRAIN, tfds.Split.TEST), \n",
+    "    with_info=True, as_supervised=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "dF8ORMt2U9lj"
+   },
+   "source": [
+    "エンコーダー（`tfds.features.text.SubwordTextEncoder`）を取得し、すこしボキャブラリを見てみましょう。\n",
+    "\n",
+    "ボキャブラリ中の \"\\_\" は空白を表しています。ボキャブラリの中にどんなふうに（\"\\_\"）で終わる単語全体と、長い単語を構成する単語の一部が含まれているかに注目してください。"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "yueILuntVEhr"
+   },
+   "outputs": [],
+   "source": [
+    "encoder = info.features['text'].encoder\n",
+    "encoder.subwords[:20]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "47oPVxgUVd7g"
+   },
+   "source": [
+    "映画のレビューはそれぞれ長さが異なっているはずです。`padded_batch` メソッドを使ってレビューの長さを標準化します。"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "0Ejbvj8mVbIE"
+   },
+   "outputs": [],
+   "source": [
+    "padded_shapes = ([None],())\n",
+    "train_batches = train_data.shuffle(1000).padded_batch(10, padded_shapes = padded_shapes)\n",
+    "test_batches = test_data.shuffle(1000).padded_batch(10, padded_shapes = padded_shapes)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "meWp84bPV0pu"
+   },
+   "source": [
+    "インポートした状態では、レビューのテキストは整数エンコードされています（それぞれの整数がボキャブラリ中の特定の単語あるいは部分単語を表しています）。\n",
+    "\n",
+    "あとの方のゼロに注目してください。これは、バッチが一番長いサンプルに合わせてパディングされた結果です。"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "T7Tef4XhV2u9"
+   },
+   "outputs": [],
+   "source": [
+    "train_batch, train_labels = next(iter(train_batches))\n",
+    "train_batch.numpy()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "zI9_wLIiWO8Z"
+   },
+   "source": [
+    "### 単純なモデルの構築\n",
+    "\n",
+    "[Keras Sequential API](../../guide/keras) を使ってモデルを定義することにします。今回の場合、モデルは「連続した Bag of Words」スタイルのモデルです。\n",
+    "\n",
+    "* 次のレイヤーは Embedding レイヤーです。このレイヤーは整数エンコードされた語彙を受け取り、それぞれの単語のインデックスに対応する埋め込みベクトルをみつけて取り出します。これらのベクトルはモデルの訓練により学習されます。このベクトルは出力配列に次元を追加します。その結果次元は `(batch, sequence, embedding)` となります。\n",
+    "\n",
+    "* 次に、GlobalAveragePooling1D レイヤーが、それぞれのサンプルについて、シーケンスの次元で平均を取り、固定長の出力ベクトルを返します。これにより、モデルは可変長の入力を最も簡単な方法で扱えるようになります。\n",
+    "\n",
```

Comment:
[question] Missing one point?

```
* This fixed-length output vector is piped through a fully-connected (Dense) layer with 16 hidden units.
```

Reply:
Well.  The dense layer with 16 unit is missing in the model code.
There is a note that says "Try deleting the Dense(16) layer,..", so this seems to be a bug in the 
original notebook.
I'd like to fix both markdown and code in Japanese translation, and send a PR to the original.

Reply:
Cool! 👍